modules/rdf.xqm

```diff
@@ -203,16 +208,122 @@ as element(rdf:RDF) {
 
   let $in_corpus := <dracon:in_corpus rdf:resource="{$collection-uri}"/>
   let $play-external-id := <owl:sameAs rdf:resource="{$wikidata-play-uri}"/>
+  
```

Comment:
Can we consistently indent by two spaces per level here and make sure individual lines are not longer than 80 characters for readability.